src/codeu/chat/client/commandline/Chat.java

```diff
@@ -47,11 +49,21 @@ public Chat(Context context) {
   // is willing to take another command, the function will return true. If
   // the system wants to exit, the function will return false.
   //
-  public boolean handleCommand(String line) {
+  public boolean handleCommand(String line) throws IOException {
```

Comment:
That's actually the commit that causes the conflict. I'm guessing you didn't do `git merge`?

In any case, you need to `git merge` now. It's not just me saying this; Github is complaining too.